src/pages/overview/pdf-electronic-seal-api/quickstarts.md

```diff
@@ -40,19 +40,19 @@ The seal field parameters are required to create a new unsigned signature field
 * **Field Name**  (*Required*): The signature field's name. This must be a non-empty string. If signature field with this field name already exist, that field will be used. 
 If it does not exist, a signature field with this field name will be created.
 * **Visibility**: Specifies whether the signature field is visible. The default value of `true` creates a visible signature.
-* **Page Number**: Specifies the page number to which the signature field should be attached. Page numbers are 1-based. The page number is only **required** if the signature field does not already exist in the pdf document. If this is specified along with the signature field then it overrides the page on which signature field is present in the document.
-* **Location**: Specifies the coordinates of the signature appearance's bounding box in default PDF user space units. The location is only **required** if the signature field does not already exist in the pdf document. If this is specified along with the signature field then it overrides the existing signature bounding box.
+* **Page Number**: Specifies the page number to which the signature field should be attached. Page numbers are 1-based. The page number is only **required** if the signature field does not already exist in the PDF document. If this is specified along with the signature field then it overrides the page on which signature field is present in the document.
+* **Location**: Specifies the coordinates of the signature appearance's bounding box in default PDF user space units. The location is only **required** if the signature field does not already exist in the PDF document. If this is specified along with the signature field then it overrides the existing signature bounding box.
 
-  * **Left**: The left x-coordinate
-  * **Bottom**: The bottom y-coordinate
-  * **Right**: The right x-coordinate
-  * **Top**: The top y-coordinate
+  * **Left**  (*Required*): The left x-coordinate
+  * **Bottom**  (*Required*): The bottom y-coordinate
+  * **Right**  (*Required*): The right x-coordinate
+  * **Top**  (*Required*): The top y-coordinate
 
 The **Location** and **Page Number** fields are **optional** if the signature field is already added in the input PDF document. To add the signature field explicitly, see [how to place a signature field in a PDF](https://www.adobe.com/sign/hub/how-to/add-a-signature-block-to-pdf).
   
 ### Seal Appearance Parameters
 
-Specifies signature field appearance parameters. These are an enum set of display items: NAME, DATE, DISTINGUISHED_NAME, LABELS and SEAL_IMAGE. Specifies the information to display in the signature. NAME and LABELS are the default values.
+Specifies the information to display in the seal. These are an enum set of display items: NAME, DATE, DISTINGUISHED_NAME, LABELS and SEAL_IMAGE. NAME and LABELS are the default values.
```

Comment:
Specifies the content to be displayed in the seal. These are an enum set of display items: NAME, DATE, DISTINGUISHED_NAME, LABELS and SEAL_IMAGE. NAME and LABELS are the default values.

Reply:
Done